server.js

```diff
@@ -0,0 +1,27 @@
+var fs = require('fs');
+var express = require('express');
+var bodyParser = require('body-parser');
+var app = express();
+var port=8080;
+
+app.use(bodyParser.json());
+
+app.use(express.static('public'));
+app.use('/src', express.static(__dirname + '/src'));
+app.set('view engine','ejs');
+app.set('views', __dirname + '/views');
+
+app.get('/', function(request,response){
+  response.render('mainMenu');
+});
+
+app.get('/menu',function(request, response){
```

Comment:
Do you really need a node backend? Given you can directly load a json file using ajax, you could have avoided all this complexity and ended up with virtually the same result - remember to keep it simple wherever possible